### PR TITLE
code reformat +  optimize imports  + few tests + object property + small bugfixes

### PR DESCRIPTION
**Pull request by dafik**

### CODE REVIEW

File: tests/CodeGen/Statement/TryCatchStatementTest.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        $this->assertCodeEqualsFile('tests/data/try_catch_empty.fixture', $tryCatch);
+
+    }
+
+    public function testTryCatch()
+    {
+        $tryCatch = new TryCatchStatement();
+        $tryCatch->tryBlock->appendLine('$x =1;');
+
+        $tryCatch->throwBlock->appendLine('throw $e');
```
Comment (c9s) [inline tests/CodeGen/Statement/TryCatchStatementTest.php]: Can we rename to catchBlock? Thanks

**Comment by c9s** [comment]
They all looking good, just one small suggestion

**Comment by c9s** [comment]
If there is a chance, see if we can separate code reformat with feature changes in different PRs. Thanks :)

File: tests/CodeGen/Statement/TryCatchStatementTest.php (hunk 1 above)
Comment (dafik) [inline tests/CodeGen/Statement/TryCatchStatementTest.php]: ok no problem

**Comment by dafik** [comment]
ok i will try separate code reformat with feature changes.

**Comment by dafik** [comment]
What should I do to complete the merge  request ?

**Comment by c9s** [comment]
Sorry for the delay! I am currently in Japan

The change looks good! Thanks!

**Pull request merged**